### DISCUSSION TITLE
Is it required to update table with hasura mutation when using subscribeToMore to subscribe new records?

### DISCUSSION THREAD

**Discussion by ngugcx** [hasura/graphql-engine#7105] [General]
In my project, the table is updated(insert/update/delete) by another service, not hasura.
Is it possible to subscribe new records by subscribeToMore or something else?

**Comment by rikinsk** [comment]
@ngugcx it is not necesssary for updates to happen via Hasura for subscriptions to work. Not sure what you mean exactly by "subscribeToMore" here. Can you elaborate?  Hasura subscriptions are live queries which return the latest result of the query periodically irrespective of how the result had been modified

**Reply by ngugcx** [reply]
https://www.apollographql.com/docs/react/data/subscriptions/#subscribing-to-updates-for-a-query
For example, when a new record is inserted, by using subscribeToMore, it only returns the new record instead of the whole result of the query.

**Comment by ngugcx** [comment]
https://github.com/hasura/graphql-engine/issues/2317
This feature request is discussing about what I want.
